### PR TITLE
absolufy-imports - No relative - PEP8

### PR DESCRIPTION
**Pull request by jsignell**
Matches https://github.com/dask/distributed/pull/5924 
Conversation in https://github.com/dask/distributed/issues/5889

### CODE REVIEW

File: dask/order.py [hunk 1]
```diff
@@ -78,8 +78,8 @@
 from collections import defaultdict, namedtuple
 from math import log
 
-from .core import get_dependencies, get_deps, getcycle, reverse_dict  # noqa: F401
-from .utils_test import add, inc  # noqa: F401
+from dask.core import get_dependencies, get_deps, getcycle, reverse_dict  # noqa: F401
+from dask.utils_test import add, inc  # noqa: F401
```
Comment (crusaderky) [inline dask/order.py:82]: Is this some legacy backwards compatibility? Could it be cleaned up?

File: dask/threaded.py [hunk 2]
```diff
@@ -13,10 +13,10 @@
...
 from threading import Lock, current_thread
 
-from . import config
-from .local import MultiprocessingPoolExecutor, get_async
-from .system import CPU_COUNT
-from .utils_test import add, inc  # noqa: F401
+from dask import config
+from dask.local import MultiprocessingPoolExecutor, get_async
+from dask.system import CPU_COUNT
+from dask.utils_test import add, inc  # noqa: F401
```
Comment (crusaderky) [inline dask/threaded.py:19]: legacy backwards compatibility? Can it be cleaned up?

**Review by crusaderky — CHANGES_REQUESTED** [review]

**Comment by jsignell** [comment]
I'm cleaning this up then going to merge.

**Comment by jsignell** [comment]
@dask/maintenance if you see lots of conflicts it's likely to be this :grimacing:

**Pull request merged**